pkg/tfvars/aws/aws.go

```diff
@@ -1,44 +1,26 @@
 package aws
 
-// Endpoints is the type of the AWS endpoints.
-type Endpoints string
-
 const (
-	// EndpointsAll represents the configuration for using both private and public endpoints.
-	EndpointsAll Endpoints = "all"
-	// EndpointsPrivate represents the configuration for using only private endpoints.
-	EndpointsPrivate Endpoints = "private"
-	// EndpointsPublic represents the configuration for using only public endpoints.
-	EndpointsPublic Endpoints = "public"
+	// DefaultVPCCIDRBlock is the default CIDR range for an AWS VPC.
+	DefaultVPCCIDRBlock = "10.0.0.0/16"
+	// DefaultRegion is the default AWS region for the cluster.
+	DefaultRegion = "us-east-1"
```

Comment:
I'd dropped these in #759; they probably snuck back in during a rebase.  Can you drop them again?